src/hscroll.story.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import { InfoWrapper } from './utils/storyComponents';
+import { decorateWithLocale } from './utils/decorators';
+import { MEDIA_SIZES } from './utils/designConstants';
+import Hscroll from './Hscroll';
+
+const itemStyle = {
+	display: 'inline-block',
+	background: 'cornsilk',
+	width: `${MEDIA_SIZES.xxl}px`,
+	height: `${MEDIA_SIZES.xl}px`,
+	outline: '1px dotted red',
+	padding: `${MEDIA_SIZES.xs}px`
+};
+
+const hscrollInlineStyle = {
+	width: '50vw'
+};
+
+const listItems = [
+	(<div style={itemStyle}>inline-block item</div>),
+	(<div style={itemStyle}>inline-block item</div>),
+	(<div style={itemStyle}>inline-block item</div>),
+	(<div style={itemStyle}>inline-block item</div>),
+	(<div style={itemStyle}>inline-block item</div>),
+	(<div style={itemStyle}>inline-block item</div>),
+	(<div style={itemStyle}>inline-block item</div>),
+	(<div style={itemStyle}>inline-block item</div>),
+	(<div style={itemStyle}>inline-block item</div>)
```

Comment:
sort of nitpick, but the parens are unnecessary and IMO noisy around inline JSX

also, trail that comma!